ST_mortality.c

```diff
@@ -885,42 +881,14 @@ void _kill_extra_growth(void) {
             if (!Species[sp]->use_me)
                 continue;
 
-            /* Annuals have already been killed and relsize for annuals set to 
-             * 0.0 in _kill_annuals */
-            if (Species[sp]->max_age == 1)
-                continue;
-            //printf("s->extragrowth kill before  = %f\n", Species[sp]->extragrowth);
-
-            /* Check that extragrowth <= s->relsize, otherwise relsize will become 
-             * negative. If not, then reset to s->relsize. If the current year 
-             * is a fire year, return, as killing of extragrowth has already occurred in Mort_EndofYear */
-            if (!ZERO(Species[sp]->extragrowth) && Globals.currYear != RGroup[rg]->killyr) {
-                Species[sp]->extragrowth = LE(Species[sp]->extragrowth, Species[sp]->relsize) ? Species[sp]->extragrowth : Species[sp]->relsize;
-
```

Comment:
@chaukap with `ZERO` now removed, the reference to it at the top and bottom of this function can be removed, i.e:

```
#define xF_DELTA (20*F_DELTA)
#define xD_DELTA (20*D_DELTA)
#define ZERO(x) \
                ( (sizeof(x) == sizeof(float)) \
                                ? ((x)>-xF_DELTA && (x)<xF_DELTA) \
                                                : ((x)>-xD_DELTA && (x)<xD_DELTA) )
```